src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -254,11 +259,16 @@ const ResearchProfile = () => {
             {!!personBackground && <Typography>{personBackground}</Typography>}
           </Box>
         )}
-        <Typography id="registration-label" variant="h2" gutterBottom sx={{ mt: '2rem' }}>
+        <Typography variant="h2" gutterBottom sx={{ mt: '2rem' }}>
           {registrationsHeading}
         </Typography>
+        <Typography>
+          <Trans t={t} i18nKey="my_page.my_profile.link_to_results_search">
+            <MuiLink component={Link} to={`/?${ResultParam.Contributor}=${encodeURIComponent(personId)}`} />
+          </Trans>
+        </Typography>
         {registrationsQuery.isPending || promotedPublicationsQuery.isPending ? (
-          <CircularProgress aria-labelledby="registration-label" />
+          <ListSkeleton minWidth={100} height={100} />
         ) : registrationsQuery.data && registrationsQuery.data.totalHits > 0 ? (
           <ListPagination
```

Comment:
Får en feil ved a11y-testing her: 
![bilde](https://github.com/user-attachments/assets/217a73f9-821e-403c-bc97-e1812bc3d42b)

Ikke relatert til denne PRen, så lager en egen PR med fiks for den, og markerer denne PRen som OK testet for UU

Reply:
Løst i egen PR her: https://github.com/BIBSYSDEV/NVA-Frontend/pull/6344